.github/workflows/lint_test.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ['3.10', '3.12']
+        python-version: ['3.7', '3.10', '3.12']
```

Comment:
Probably worth adding a comment here explaining 3.7 is to support entreprise customers on legacy Python version (not mentionning the customer explicitely in the comment)

Reply:
Fixed